### ISSUE TITLE
Header: custom headerTitle element breaks layout on iOS

### ISSUE THREAD

**Issue by EmielM** [react-community/react-navigation#1018]
Hey,

When using custom `headerTitle`s, the first transition of the `Header` in a `StackNavigator` on iOS always results in the previous title and buttons showing through (after the transition is complete):

In action:
https://giphy.com/gifs/3ohzdMrhQ43CbNAjWo

Code used to generate this is the most simple `StackNavigator` with two `<View />` components as `headerTitle` (bgcolor'ed green and yellow) for both the home screen and the profile screen.

I can imagine it having to do something with the `Header` component [doing an early exit for custom headerTitle components](https://github.com/react-community/react-navigation/blob/master/src/views/Header.js#L100) and not executing the onLayout callback.

This is terribly annoying, as it really breaks the first-time experience for users. I've had this issue since the earliest versions of this component.

**Comment by grabbou** [comment]
Thanks for a detailed description! This is a duplicate of #229.

tl;dr

This is a bug in React Native `Animated` and awaits fix in React Native repository.
